helm/chart/router/README.md

```diff
@@ -70,8 +70,6 @@ helm show values apollographql/router
 | podSecurityContext | object | `{}` |  |
 | replicaCount | int | `1` |  |
 | resources | object | `{}` |  |
-| rhai | object | `{"input_file":""}` | If using rhai, specify the location of your input file |
-| rhai.input_file | string | `""` | input rhai file, contents will be stored in a ConfigMap |
```

Comment:
Is this a drive-by fix for https://github.com/apollographql/router/pull/1917?

Reply:
Kind of. The README.md is generated by `helm-doc` and so the change to the README.md file reflects the change which was the fix in #1917.